crates/cast/tests/e2e/account/delete.rs

```diff
@@ -0,0 +1,243 @@
+    let stdout_str =
+        std::str::from_utf8(&out.stdout).expect("failed to convert command output to string");
+    let _ = std::fs::remove_file("temp_accounts1.json");
+    assert!(stdout_str.contains("Account successfully removed"));
+}
+
+#[tokio::test]
+pub async fn test_happy_case_with_remove_profile() {
+    // Creating dummy accounts and scarb.toml test file
+    let mut json_data = indoc! {r#"
+    {
+        "alpha-goerli": {
+            "__default__": {
+                "private_key": "0x421b62d2cba1fd39798d719a6cee5f599afc79b0bacbea50f76215057c068dd",
+                "public_key": "0x12d3ad59161fd2a72d5bc8501bb2f2ca1acd34706d2dfa31a90aadb4b41e050",
+                "address": "0x20f8c63faff27a0c5fe8a25dc1635c40c971bf67b8c35c6089a998649dfdfcb"
+            },
+            "user0": {
+                "private_key": "0x1e9038bdc68ce1d27d54205256988e85",
+                "public_key": "0x2f91ed13f8f0f7d39b942c80bfcd3d0967809d99e0cc083606cbe59033d2b39",
+                "address": "0x4f5f24ceaae64434fa2bc2befd08976b51cf8f6a5d8257f7ec3616f61de263a"
+            },
+            "user1": {
+                "private_key": "0x55ae34c86281fbd19292c7e3bfdfceb4",
+                "public_key": "0xe2d3d7080bfc665e0060a06e8e95c3db3ff78a1fec4cc81ddc87e49a12e0a",
+                "salt": "0x14b6b215424909f34f417ddd7cbaca48de2d505d03c92467367d275e847d252",
+                "address": "0x112b61669a84940af30544786a6b615755b192173c4665a7d38fb3aa0541cac",
+                "deployed": true
+            }
+        }
+    }
+    "#};
+
+    let mut file =
+        File::create("temp_accounts2.json").expect("Could not create temporary accounts file!");
+    file.write_all(json_data.as_bytes())
+        .expect("Could not write temporary testing accounts");
+    let _ = file.flush();
+
+    json_data = indoc! {r#"
+    [dependencies]
+    starknet = "1.1.1"
+
+    [dependencies.snforge_std]
+    git = "https://github.com/foundry-rs/starknet-foundry"
+    tag = "v0.6.0"
+
+    [package]
+    name = "mypackage"
+    version = "0.1.0"
+
+    [[target.starknet-contract]]
+    casm = true
+
+    [tool.sncast]
+    url = "http://127.0.0.1:5050/rpc"
+
+    [tool.sncast.user0]
+    account = "a4"
+    accounts-file = "./myfile.json"
+    url = "http://127.0.0.1:5050/rpc"
+
+    [tool.sncast.user1]
+    account = "a4"
+    accounts-file = "./myfile.json"
+    url = "http://127.0.0.1:5050/rpc"
+    "#};
+
+    let mut file = File::create("Scarb.toml").expect("Could not create temporary scarb file!");
+    file.write_all(json_data.as_bytes())
+        .expect("Could not write temporary scarb accounts");
+    let _ = file.flush();
+
+    // Now delete dummy account from accountd file and his profile from scarb
+    let args = vec![
+        "--url",
+        "http://127.0.0.1:5050/",
+        "--accounts-file",
+        "temp_accounts2.json",
+        "account",
+        "delete",
+        "--name",
+        "user1",
+        "--network",
+        "alpha-goerli",
```

Comment:
it would be easier for users to provide network with their user-friendly names - eg `testnet` or `mainnet`.

Reply:
yes but accounts file use this notation for networks so i think it would be better to keep this names. Also i can reuse chain_id_to_network_name function which already exists and don't have to add another mapping for testnet/mainnet. Please tell me if you still prefers to change to testnet/mainnet names or we are ok with alpha-goerli, etc?

Reply:
I think the alpha-goerli-like names are fine